server/src/email-password/signup.ts

```diff
@@ -36,7 +34,6 @@ export default async (event: FunctionEvent<EventData>) => {
         }
 
         // create password hash
-        const salt = bcrypt.genSaltSync(SALT_ROUNDS);
         const hash = await bcrypt.hash(password, SALT_ROUNDS);
```

Comment:
der salt ist hier jetzt immer "10" (`.hash` erwartet nämlich als zweiten Parameter den salt: https://www.npmjs.com/package/bcryptjs).
Ist natürlich auch ne Möglichkeit sich das sparen vom salt zu speichern :D

(ich glaube das bei graphcool scheinen richtige Sicherheitsprofis zu sein 🙈 )

Reply:
ok, also salt brauch man tatsächlich nicht separat speichern, der ist per trennzeichen im hash ergebnis eingebettet – trotzdem sollte er vllt nicht einfach immer "10" sein xD

Reply:
Ne, hab ich auch erst gedacht, aber wenn der zweite parameter vom typ `number` ist, ist das die Menge der SALT_ROUNDS.

Reply:
okay, vergessen wir das – wenn der zweite Parameter type number ist, wird der salt in der Länge generiert ...